app/assets/stylesheets/application.scss

```diff
@@ -19,8 +19,13 @@
 @import "helpers/content-bottom-margin";
 @import "helpers/publisher-metadata-with-logo";
 
-// components
+// Components from this application
 @import 'components/*';
+
+// Components from govuk_publishing_components gem
+@import 'components/fieldset';
```

Comment:
Using the same namespace is confusing isn't it.
I would have expected `govuk_publishing_components/fieldset`

Reply:
Yeah very, we could consider moving the components, for now this comment should help somewhat.

cc @andysellick

Reply:
Yeah, govuk_publishing_components/component-name would be good. Firebreak? :)

Reply:
Namespace problem being looked at in https://github.com/alphagov/govuk_publishing_components/pull/136